src/Dropdown.js

```diff
@@ -119,22 +120,46 @@ export default class Dropdown extends Component {
 
   render() {
     const { handleMouseEnter, handleMouseLeave, handleClick } = this
-    const { mode, children, component } = this.props
+    const { mode, children, component, link } = this.props
     const { isOpen } = this.state
-    const className = cx(this.props.className, {
+    const className = cx('uk-button uk-button-default', {
       'uk-open': isOpen,
     })
-    const DropdownProps = {
+
+    const dropdownClassName = cx('uk-dropdown uk-dropdown-bottom-left', {
```

Comment:
The default classname here `.uk-dropdown .uk-dropdown-bottom-left` must be made configurable using a prop.